pkg/structs/flamebearer/html.go

```diff
@@ -0,0 +1,45 @@
+package flamebearer
+
+import (
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"io"
+	"net/http"
+)
+
+// FlameberarerToHTML converts and writes a flamebearer into HTML
+func FlameberarerToHTML(fb *FlamebearerProfile, dir http.FileSystem, w io.Writer) error {
```

Comment:
```suggestion
func FlameberarerToStandaloneHTML(fb *FlamebearerProfile, dir http.FileSystem, w io.Writer) error {
```
I would suggest for consistency to use the `standalone` terminology. The reason is that it's not that we are really converting a flamegraph to HTML, we are converting a flamegraph to a "standalone HTML", which has different attributes. 

A simple flame->HTML conversion could be just the flamegraph itself for example, one possible use is for a hypothetical embed widget.

Where a standalone HTML (should) contain more stuff, like a nice header, footer, links to other places etc.

Reply:
Good point. Changed in https://github.com/pyroscope-io/pyroscope/pull/728/commits/a04b3607757e541381367874c86a0d62284a8aa7